x/did/internal/types/events.go

```diff
@@ -0,0 +1,14 @@
+package types
+
+const (
+	EventTypeAddDidDoc     = "create_did_doc"
+	EventTypeAddCredential = "credential"
+
+	AttributeKeyDid        = "did"
+	AttributeKeyPubKey     = "pub_key"
+	AttributeKeyCredType   = "credType"
```

Comment:
```suggestion
	AttributeKeyCredType   = "cred_type"
```

Reply:
AttributeKeyCredType   = "credType" had been changed to AttributeKeyCredType     = "cred_type" . hope this is fine